### ISSUE TITLE
Add more ideologies

### ISSUE THREAD

**Issue by ByerCame0061** [8values/8values.github.io#72]
Geolibertarianism/Bleeding-Heart Libertarianism
Economy: 30
Diplomacy: 60
Government: 80
Society: 80

National Anarchism
Economy: 60
Diplomacy: 20
Government: 100
Society: 20

Anarcho-Primitivism
Economy: 100
Diplomacy: 60
Government: 100
Society: 0

Christian Democracy
Economy: 50
Diplomacy: 50
Government: 30
Society: 20

Libertarian Conservatism
Economy: 20
Diplomacy: 40
Government: 50
Society: 30

Paleoconservatism
Economy: 30
Diplomacy: 30
Government: 50
Society: 10

Neoreactionism
Economy: 0
Diplomacy: 20
Government: 20
Society: 0

Greenism
Economy: 70
Diplomacy: 80
Government: 70
Society: 80

National Liberalism
Economy: 50
Diplomacy: 20
Government: 50
Society: 40

Anarcho-Collectivism
Economy: 90
Diplomacy: 50
Government: 100
Society: 90

Minarchism
Economy: 10
Diplomacy: 50
Government: 90
Society: 50

Voluntaryism
Economy: 5
Diplomacy: 50
Government: 95
Society: 50

Paleolibertarianism
Economy: 80
Diplomacy: 40
Government: 60
Society: 20

Strasserism
Economy: 60
Diplomacy: 0
Government: 0
Society: 5

Left-Wing Populism
Economy: 70
Diplomacy: 20
Government: 60
Society: 40

**Comment by brmbrmcar** [comment]
Just adding more ideologies is a really bad idea, especially as these are basically all covered by something else. Until we fix the way that ideologies are calculated altogether we should avoid any ideologies with really tiny differences that the test doesn't test for.

**Comment by hsoj95** [comment]
Yeah, I do agree with this. Also, I think @TristanBomb talked about wanting to overhaul the way the ideologies system works. So any additions should probably be put on hold for the moment!
